nvme-print.c

```diff
@@ -2934,9 +2940,38 @@ void json_nvme_zns_id_ns(struct nvme_zns_id_ns *ns,
 	json_free_object(root);
 }
 
+static void show_nvme_id_ns_zoned_zoc(__le16 ns_zoc)
+{
+	__u16 zoc = le16_to_cpu(ns_zoc);
+	__u8 rsvd = (zoc & 0xfc) >> 2;
+	__u8 ze = (zoc & 0x2) >> 1;
+	__u8 vzc = zoc & 0x1;
+	if (rsvd)
+		printf("  [7:2] : %#x\tReserved\n", rsvd);
+	printf("  [1:1] : %#x\tZone Active Excursions: %s\n",
+		ze, ze ? "Yes" : "No");
```

Comment:
Please modify to

"Yes (Host support required)" for both ZAE and VZC.

Reply:
@MatiasBjorling it does not always required.

Reply:
Why is it not always required? The host will lose data if it does not implement support.

Reply:
About. VZC
If the Variable Zone Capacity bit is set to ‘1’ in the Zone Operation Characteristics field
in the Zoned Namespace Command Set specific Identify Namespace data structure
(refer to Figure 8), then the zone capacity may change upon successful completion of
a Zone Management Send command specifying the Zone Send Action of Reset Zone
(refer to section 4.3.1.4).

**It just means can be modified zone cap by zome mgmt send. If the host manages the capacity with kinda file system, then support required. But not all of host manages this. It means, its only required if its been handled by host.** 

About. ZAE
If set to ‘1’, then a controller may transition a zone in the ZSIO:Implicitly Opened state, the ZSEO:Explicitly Opened state, or the ZSC:Closed state to the ZSF:Full state due to a
vendor specific excursion event.

**It is also same as VZC, if all of zone status is managed by host, then it is required. But incase of implictly using zns, It is not required.**

Reply:
Any host, that uses a drive that has a Zone Operation Characteristic quirks, i.e., is "1", must implement support. If not, the host is broken and will get errors if it is specifically written to understand this behavior.

Reply:
@MatiasBjorling 
spec says "due to a vendor specific excursion event", so it is not always...
It might be needed for some, but not always....
By device behavior, it won't be needed.